Documentation/NextSteps/Index.rst

```diff
@@ -8,6 +8,7 @@ Next steps
 ==========
 
 If your activities are not limited to editorial work, you may want to further
-learn about TYPO3 CMS by following the :doc:`TypoScript in 45 minutes Tutorial
```

Comment:
Do you remove the reference to the "TypoScript in 45 minutes" tutorial by intention?

Reply:
Yes, the sitepackage contains the complete Information on how to start a site package. You wouldn't get ppl started by teaching them TypoScript form the scratch today...